nova/tests/unit/virt/vmwareapi/fake.py

```diff
@@ -813,10 +835,9 @@ def __init__(self, name="ha-host", connected=True, ds_ref=None,
         self.set("config.network.pnic", net_info_pnic)
         self.set("connected", connected)
 
-        if _db_content.get("Network", None) is None:
+        if not _db_content.get("Network", None):
```

Comment:
Except testing for True is "more" correct here, since we also want to create a network if there is an empty dict, otherwise the code in the next line fails. The `get(..., None) follows the pattern of the existing code. But I'll change it (everywhere).

I'll encapsulate the logic into a separate function `_no_objects_of_type`, so it becomes clearer.